src/lib.rs

```diff
@@ -499,6 +499,15 @@ pub struct Config {
     max_send_udp_payload_size: usize,
 }
 
+// See https://quicwg.org/base-drafts/draft-ietf-quic-transport.html#section-15
+fn is_reserved_version(version: u32) -> bool {
+    version == 0x00000000 ||
```

Comment:
The spec is a bit terse, but I think what it's saying is that `0x00000000` is reserved only for use within [version negotiation packets](https://quicwg.org/base-drafts/rfc9000.html#name-version-negotiation-packet), so I don't think we should allow users to configure this version.